cognee/api/v1/ui/ui.py

```
@@ -327,55 +425,111 @@ def prompt_user_for_download() -> bool:
+                    f"{mcp_port}:8000",
+                    "--rm",
+                    "--env-file",
+                    env_file,
+                    "-e",
+                    "TRANSPORT_MODE=sse",
+                    "cognee/cognee-mcp:main",
+                ],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                preexec_fn=os.setsid if hasattr(os, "setsid") else None,
+            )
```

Pylint (3.3.8)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)